### PR TITLE
Updating YesSql packages

### PR DESCRIPTION
**Pull request by sebastienros**

### CODE REVIEW

File: src/OrchardCore.Modules/Orchard.Indexing/IndexingTaskManager.cs [hunk 1]
```diff
@@ -80,7 +80,7 @@ public void Dispose()
             }
 
             var connection = _store.Configuration.ConnectionFactory.CreateConnection();
-            await connection.OpenAsync();
+            connection.Open();
```
Comment (Jetski5822) [inline src/OrchardCore.Modules/Orchard.Indexing/IndexingTaskManager.cs]: why the removal of async?

File: src/OrchardCore/Orchard.Data/Migration/DataMigrationManager.cs [hunk 2]
```diff
@@ -151,89 +152,87 @@ public class DataMigrationManager : IDataMigrationManager
             // apply update methods to each migration class for the module
             foreach (var migration in migrations)
             {
-                _session.ExecuteMigration(schemaBuilder =>
-                {
-                    migration.SchemaBuilder = schemaBuilder;
+                var schemaBuilder = new SchemaBuilder(_session);
```
Comment (Jetski5822) [inline src/OrchardCore/Orchard.Data/Migration/DataMigrationManager.cs]: I prefered the ExecuteMigration method

File: src/OrchardCore/Orchard.Data/Migration/DataMigrationManager.cs [hunk 3]
```diff
@@ -151,89 +152,87 @@ public class DataMigrationManager : IDataMigrationManager
...
+
+                    dataMigrationRecord.Version = current;
+                }
+                catch (Exception ex)
+                {
+                    if (ex.IsFatal())
                     {
-                        // Persist data migrations
-                        _session.Save(_dataMigrationRecord);
+                        throw;
```
Comment (Jetski5822) [inline src/OrchardCore/Orchard.Data/Migration/DataMigrationManager.cs]: we dont cancel the session if the exception is fatal?

File: src/OrchardCore/Orchard.Data/ServiceCollectionExtensions.cs [hunk 4]
```diff
@@ -12,9 +12,13 @@
...
 using Orchard.Environment.Shell;
-using YesSql.Core.Indexes;
-using YesSql.Core.Services;
+using YesSql.Indexes;
 using YesSql.Storage.Sql;
+using YesSql;
+using YesSql.Provider.MySql;
+using YesSql.Provider.Sqlite;
+using YesSql.Provider.SqlServer;
+using YesSql.Provider.PostgreSql;
```
Comment (Jetski5822) [inline src/OrchardCore/Orchard.Data/ServiceCollectionExtensions.cs]: YeslSql.Provider

All the extensions methods to add these providers should be at the root namespace. Not custom ones.

File: src/OrchardCore.Modules/Orchard.Indexing/IndexingTaskManager.cs (hunk 1 above)
Comment (sebastienros) [inline src/OrchardCore.Modules/Orchard.Indexing/IndexingTaskManager.cs]: It's not available on the interface, just on the concrete type.

File: src/OrchardCore/Orchard.Data/Migration/DataMigrationManager.cs (hunk 2 above)
Comment (sebastienros) [inline src/OrchardCore/Orchard.Data/Migration/DataMigrationManager.cs]: separation of concerns, this way the method doesn't have to be on ISession, which mean a lot less dependencies.

File: src/OrchardCore/Orchard.Data/ServiceCollectionExtensions.cs (hunk 4 above)
Comment (sebastienros) [inline src/OrchardCore/Orchard.Data/ServiceCollectionExtensions.cs]: Meh. will change it in YesSql when I am bored.

**Pull request merged**